nixos/modules/services/monitoring/scrutiny.nix

```diff
@@ -110,7 +110,11 @@ in
       };
 
       collector = {
-        enable = mkEnableOption "the Scrutiny metrics collector";
+        enable = lib.mkOption {
+          type = bool;
+          default = cfg.enable;
+          description = "Enable the Scrutiny metrics collector";
+        };
```

Comment:
```suggestion
        enable = mkEnableOption "the Scrutiny metrics collector" // {
          default = cfg.enable;
          defaultText = lib.literalExpression "config.services.scrutiny.enable";
        };
```

Reply:
Oh that's interesting. I haven't seen the double slash before. What does it do exactly? Allow you to modify the generated set?

Reply:
https://nix.dev/manual/nix/2.18/language/operators#update